### PR TITLE
Adding test DB initialisation script

### PR DESCRIPTION
**Pull request by DeltaO3**
Instructions to run the script included at the bottom of the readme. Simple script to initialise database with test data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,10 +119,19 @@ python -m app.createadmin
...
+```shell
 python -m app.utilities
 ```
+
+# Initialising test data
+
+With an empty db file (delete app.db, create fresh file with `flask db upgrade`), run:
+
+```
+python - m app.testdb
```
Comment (ipr17) [inline README.md:136]: tiny tiny thing, there is a typo on this line. I mean its obvious what you mean (-m not - m) but might be best to ensure the documentation is correct. The rest looks great though :)

**Review by ipr17 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (DeltaO3) [reply README.md:136]: lmao how did i miss this, good catch

**Review by livvy10 — CHANGES_REQUESTED** [review]
Mine crashed... I think the issue is potentially just a typo for the add unit to facilitator 44456666 but there is no user with that uwaID, instead it is probably meant to be the user  44446666? As far as I can tell I don't think addunittofacilitator automatically makes a user if they don't already exist

 File "/home/olivia/UWAttend---3200_27/app/testdb.py", line 21, in <module>
    AddUnitToFacilitator(44456666, unit1)
  File "/home/olivia/UWAttend---3200_27/app/database.py", line 161, in AddUnitToFacilitator
    user.unitsFacilitate.append(unit)
AttributeError: 'NoneType' object has no attribute 'unitsFacilitate'

**Comment by DeltaO3** [comment]
Should be fixed, dunno how 2 typos passed through this

**Review by livvy10 — APPROVED** [review]
works for me!

**Pull request merged**